data/helpers/projectsDb.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../dbConfig.js');
+
+module.exports = {
+    getProjects,
+    getProjectbyId,
+    addProject,
+    deleteProject,
+    updateProject
+};
+
+function getProjects() {
+    return db('projects')
+}
+
+function getProjectbyId(id) {
+    return db('projects')
+        .where({ id })
+        .first();
+}
+
+function addProject(project) {
+    return db('projects')
+        .insert(project)
+        .then(ids => {
+            return getProject(ids[0])
+        });
+}
+
+function deleteProject(id) {
+    return db('projects')
+        .where({ id })
+        .del()
+}
+
+function updateProject(id, changes) {
+    return db('projects')
+        .where({ id })
+        .update(changes)
+} 
```

Comment:
https://github.com/josephpak/Sprint-Challenge-RDBMS/blob/ebf5beef5318ba6e51d12ec88fa291eaf40d26dd/data/helpers/actionsDb.js#L20-L44

✔️ clean, concise syntax and proper implementation of Knex.js library for DB CRUD operations